blueprints/data-solutions/data-platform-foundations/03-orchestration.tf

```diff
@@ -66,6 +73,13 @@ module "orch-project" {
     "roles/storage.objectAdmin" = [
       module.orch-sa-cmp-0.iam_email,
       "serviceAccount:${module.orch-project.service_accounts.robots.composer}",
+      "serviceAccount:${module.orch-project.service_accounts.robots.cloudbuild}",
```

Comment:
can you update the IAM.md file with the updated roles?

Fabric has a tool to generate the file from the Terraform state file.

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/tools/state_iam.py

Remember to use the `--prefix` parameter to remove the prefix.

Reply:
Is there a way to easily visualise the diff between the existing roles? even better if the tables can be somehow generated ?